appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/reject/reject.controller.js

```diff
@@ -31,7 +31,7 @@ export const renderSelectReason = async (request, response) => {
 		const mappedRejectionReasons = mapRejectionReasonOptionsToCheckboxItemParameters(
 			currentComment,
 			rejectionReasons,
-			errors
+			errors?.['']
 				? { optionId: parseInt(errors[''].value.rejectionReason), message: errors[''].msg }
```

Comment:
@Si3rr4wow 

It was bamboozling to figure out 😄 

It's because [here](https://github.com/Planning-Inspectorate/appeals-back-office/blob/3899e75a30b9085ebec03f122aea48b6891a00da/appeals/web/src/server/lib/validators/checkbox-text-items.validator.js#L12) the call to `body()` is a check against the whole request body, not a specific key. So when it finds an error it places it under the `['']` key.

In other places it's called like `body('someKey')` and then the `someKey` would also appear in the errors object.